src/scss/colorways.scss

```
@@ -1570,3 +1570,36 @@
 .mt3-susuwatari-ortho-kb {
   // placeholder
 }
+
+.dcs-midnight-kb {
+  // placeholder
+}
+
+.dcs-midnight-key {
+  background: $color-dcs-abs-mid-alpha;
+  color: $color-dcs-abs-mid-accent-cyan;
```

The following suggestions swap out the color codes for the existing codes (they're the exact colors in each case).

```suggestion
  background: $color-sp-abs-GB;
  color: $color-sp-abs-VCO;
```